static/app/views/organizationStats/usageStatsProjects.tsx

```diff
@@ -250,11 +250,18 @@ class UsageStatsProjects extends DeprecatedAsyncComponent<Props, State> {
         onClick: () => this.handleChangeSort(SortBy.FILTERED),
       },
       {
-        key: SortBy.DROPPED,
-        title: t('Dropped'),
+        key: SortBy.RATE_LIMITED,
```

Comment:
We're opting not to use the term "dropped" because it could easily be confused with `client_discard` Customers might ask, "What's the difference between `dropped` and `client discarded`?" We could explain that `client_discard` refers to transactions discarded within the SDK, whereas `dropped` refers to transactions discarded on the server. However, this explanation might get too technical.